jwst/datamodels/schemas/keyword_nframes.schema.yaml

```diff
@@ -0,0 +1,12 @@
+type: object
+properties:
+  meta:
+    type: object
+    properties:
+      exposure:
+        type: object
+        properties:
+          nframes:
+            title: Number of frames per group
+            type: number
```

Comment:
The current datamodels infrastructure will allow "number" to be either integer or float, but given that this keyword value should always be a whole number, I would say we could explicitly declare it to be integer (that's how it's written in FITS headers).